air_firmware/main/main.cpp

```diff
@@ -690,7 +690,9 @@ esp_err_t set_wifi_fixed_rate(WIFI_Rate value)
         WIFI_PHY_RATE_MCS7_LGI,
         WIFI_PHY_RATE_MCS7_SGI,
     };
-    esp_err_t err = esp_wifi_internal_set_fix_rate(ESP_WIFI_IF, true, (wifi_phy_rate_t)rates[(int)value]);
+
+    esp_err_t err = esp_wifi_config_80211_tx_rate(ESP_WIFI_IF, (wifi_phy_rate_t)rates[(int)value]);
```

Comment:
The rates in that array were hand-tuned for the `esp_wifi_internal_set_fix_rate` API. Not sure if they map well to the `esp_wifi_config_80211_tx_rate` API. I used wireshark to capture the packets and validate the rates for each of the setting.

Reply:
debug findings:  please check and compare under one factor only. 
Because I made a mess here.

a) sdkconfig  will cause "interface invalid"
b) no RF packet send might be caused by "esp_wifi_internal_set_fix_rate"